### ISSUE TITLE
Add option the remove armor during a game , so you can  custom armor

### ISSUE THREAD

**Issue by AIGLE25** [andrei1058/BedWars1058#736]
**Is your feature request related to a problem? Please describe.**
Hello , i need a option to wear custom armor beacause i'm stck with a bad armor , in my game mode (Lucky Wars) we only need to respawn in our base if the bed didn't get broke , it's all , we can get a lot of items and wear any armors , but in the plugin ,i am stuck with a leather armor.

![image](https://user-images.githubusercontent.com/68445450/227723759-043419f2-0286-451e-a56c-a362fb2bac6b.png)

**Describe the solution you'd like**
Put a option in config file to disable auto armor  

**Describe alternatives you've considered**

**Additional context**

**Comment by tomkeuper** [comment]
In my opinion this is outside the scope of bedwars. 
If you want to create a different gamemode based of BedWars you should fork this project and change the source code.
Or still use the shop thats already build into bedwars?

**Comment by AIGLE25** [comment]
> In my opinion this is outside the scope of bedwars. If you want to create a different gamemode based of BedWars you should fork this project and change the source code. Or still use the shop thats already build into bedwars?

yes but i don't know how to code , and we can get amor by opening luckyblocks

**Comment by tomkeuper** [comment]
You could ask a dev to change that for you.
The problem is that this will require more changes than just disabling auto armor.

**Comment by AIGLE25** [comment]
> 

oh yes , for my gamemode , you can check this video at 12:23

https://youtu.be/_N3Xw2yJVTs

**Comment by AIGLE25** [comment]
i think i just need an option here like "enable-auto-armor: false"
![image](https://user-images.githubusercontent.com/68445450/229350297-aeb5078c-22c6-499a-bea9-c7eeab0feb8c.png)

**Comment by andrei1058** [comment]
You will end up asking more and more customisation like "well, now I need you to block other teams from wearing my team's dropped armor" ecc

**Comment by AIGLE25** [comment]
"well, now I need you to block other teams from wearing my team's dropped armor"  nope i don't need it  , but i am sorry for asking a lot of features.

**Comment by andrei1058** [comment]
That makes more sense with the new 1.20 armor update
